west.yml

```diff
@@ -1,7 +1,7 @@
 manifest:
   projects:
     - name: zephyr
-      revision: d51a67b7c098c8bb5479a7b87f05dc40fb76e065
+      revision: 4f8d78ceeb436e82f528511998515f6fc137c6cd
```

Comment:
Can we put `v3.0.0` here instead?

Reply:
Yes, we can. Checked NCS to see whether they use tags there. They use git tags there, so no reason we should do it differently. Will update this PR.

Reply:
Would be good to include a CI check that runs on each root README.rst update, with something like `git merge-base --is-ancestor <commit-ref-from-device-matrix> <ci-commit-ref>` just to check we didn't merge with wrong commit id / tag (this happened already when using GitHub Merge button, as GitHub always does rebase even if fast-forward would work).